docs/running-clusterfuzzlite/prow.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: default
+parent: Running ClusterFuzzLite
+grand_parent: ClusterFuzzLite
+title: Prow
+nav_order: 3
+permalink: /running-clusterfuzzlite/prow/
+---
+# Prow
+{: .no_toc}
+
+- TOC
+{:toc}
+---
+
+This document explains how to set up ClusterFuzzLite on
+[Prow](https://github.com/kubernetes/test-infra/tree/master/prow#readme).
+This document assumes the reader has set up Prow on their repo.
+
+First, you must
+[create a Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/creating-buckets).
+This will be refered to as
+`<your-cloud-bucket>` in the config examples in this document. If
+The bucket will be used to store crashes, builds, corpora, and coverage reports.
+
+Then you must create a service account with access to both `<your-cloud-bucket>` and the Prow Results Bucket
+To do this
+  1) Create a Cloud Service account in your Project
+
+    gcloud beta iam service-accounts create `<SA_NAME>` --project=`<PROJECT_ID>` --description=`<SA_DESCRIPTION>` --display-name=`<SA_DISPLAY_NAME>`
+
+  2) Give the Cloud Service account access to BOTH the k8s Prow Results Bucket AND `<your-cloud-bucket>` using the command
+
+    gsutil iam ch "serviceAccount:${SA}:roles/storage.objectAdmin" "${BUCKET}"
```

Comment:
Should `$SA` be SA_NAME or something else?

Reply:
Yea. <SA_NAME> fits what we have been doing in the rest of the docs